src/app/containers/Figure/index.test.jsx

```diff
@@ -64,7 +65,7 @@ describe('Figure', () => {
   );
 
   shouldMatchSnapshot(
-    'should render can AMP image with aption text',
+    'should render can AMP image with caption text',
```

Comment:
```suggestion
    'should render an AMP image with caption text',
```
(good job on catching that typo! I think the c from caption ending up before `an` here, so we can remove it maybe?)